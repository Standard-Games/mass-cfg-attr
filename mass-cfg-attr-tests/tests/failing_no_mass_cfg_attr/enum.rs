#![deny(unused_must_use, deprecated)]
#![allow(path_statements)] // disable warning for test

#[must_use]
enum Enum {
    #[deprecated]
    NotOk,

    Ok,
}

fn main() {
    // Violates unused_must_use and deprecated without mass_cfg_attr
    Enum::NotOk;
}
