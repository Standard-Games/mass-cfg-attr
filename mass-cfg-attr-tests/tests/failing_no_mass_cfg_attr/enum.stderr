error: use of deprecated unit variant `Enum::NotOk`
  --> tests/failing_no_mass_cfg_attr/enum.rs:14:11
   |
14 |     Enum::NotOk;
   |           ^^^^^
   |
note: the lint level is defined here
  --> tests/failing_no_mass_cfg_attr/enum.rs:1:26
   |
1  | #![deny(unused_must_use, deprecated)]
   |                          ^^^^^^^^^^

error: unused `Enum` that must be used
  --> tests/failing_no_mass_cfg_attr/enum.rs:14:5
   |
14 |     Enum::NotOk;
   |     ^^^^^^^^^^^^
   |
note: the lint level is defined here
  --> tests/failing_no_mass_cfg_attr/enum.rs:1:9
   |
1  | #![deny(unused_must_use, deprecated)]
   |         ^^^^^^^^^^^^^^^
